demo/helpers/demo-list-helper.html

```diff
@@ -0,0 +1,44 @@
+<script>
+	window.Cosmoz = window.Cosmoz || {};
+
+	Cosmoz.Demo = Cosmoz.Demo || {};
+
+	function getRandomInt(min, max) {
+		return Math.floor(Math.random() * (max - min + 1)) + min;
+	}
+
+	Cosmoz.Demo.generateListDemoData = function (groupsMin, groupsMax, itemsMin, itemsMax) {
```

Comment:
nitpicking - replace `function` for short hand?

Reply:
Replaced it.